e2e-test/example_test.go

```diff
@@ -0,0 +1,229 @@
+package e2e_test
```

Comment:
example_test.go as file name is confusing...
Maybe just e2e_test.go or appconfig_test.go?